Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -33,27 +34,37 @@ public virtual ITransport transport
         }
 
         // configuration
-        [FormerlySerializedAs("m_NetworkAddress")] public string networkAddress = "localhost";
         [FormerlySerializedAs("m_DontDestroyOnLoad")] public bool dontDestroyOnLoad = true;
         [FormerlySerializedAs("m_RunInBackground")] public bool runInBackground = true;
         [FormerlySerializedAs("m_ShowDebugMessages")] public bool showDebugMessages;
-        [FormerlySerializedAs("m_PlayerPrefab")] public GameObject playerPrefab;
-        [FormerlySerializedAs("m_AutoCreatePlayer")] public bool autoCreatePlayer = true;
-        [FormerlySerializedAs("m_PlayerSpawnMethod")] public PlayerSpawnMethod playerSpawnMethod;
+
+        [Scene]
         [FormerlySerializedAs("m_OfflineScene")] public string offlineScene = "";
```

Comment:
did you try using 'Scene offlineScene'? Or do we absolutely need a custom attribute?